src/main/java/com/thealgorithms/sorts/BubbleSort.java

```diff
@@ -16,11 +16,14 @@ class BubbleSort implements SortAlgorithm {
      * @param <T> the type of elements in the array.
      * @return the sorted array.
      */
+    
     @Override
     public <T extends Comparable<T>> T[] sort(T[] array) {
-        for (int i = 1, size = array.length; i < size; ++i) {
-            boolean swapped = false;
-            for (int j = 0; j < size - i; ++j) {
+        int n = array.length;
+        boolean swapped;
+        for (int i = 1; i < n; ++i) {
+            swapped = false;
+            for (int j = 0; j < n - i; ++j) {
```

Comment:
This change does not make any sense: the scope of the variable `swapped` is bigger. Moreover calling the `array.length` is just querying the `length` field - introducing the new variable does not make this function any faster.